qa/rpc-tests/fundrawtransaction.py

```diff
@@ -28,7 +28,10 @@ def setup_network(self, split=False):
 
     def run_test(self):
         print "Mining blocks..."
-        feeTolerance = Decimal(0.00000002) #if the fee's positive delta is higher than this value tests will fail, neg. delta always fail the tests
+
+        # if the fee's positive delta is higher than this value tests will fail, neg. delta always fail the tests
+        #            = 2 bytes * minRelayTxFeePerByte
+        feeTolerance = 2 * Decimal('0.00005000')/1000
```

Comment:
The right way is to fix the expected fees, not change the tolerance to nno longer catch errors.